include/irr/core/containers/LRUCache.h

```diff
@@ -167,14 +167,6 @@ class LRUCache : private impl::LRUCacheBase<Key,Value,MapHash,MapEquals>
 		}
 
 		//get the value from cache at an associated Key, or nullptr if Key is not contained within cache. Does not alter the value use order
-		inline Value* peek(const Key& key)
-		{
-			uint32_t i = common_peek(key);
-			if (i!=invalid_iterator)
-				return &(m_list.get(i)->data.second);
-			else
-				return nullptr;
-		}
```

Comment:
dont remove the non-const `peek`, because then we'll be never able to peek a `Value*` and access it to modify the value.